internal_packages/message-list/lib/message-timestamp.cjsx

```diff
@@ -1,5 +1,6 @@
 _ = require 'underscore'
 moment = require 'moment-timezone'
+moment = require "moment"
```

Comment:
This looks like the variable is overriding the previous one. However, we need `moment-timezone` instead of just `moment`. Can you remove the duplicate one here.